docs/docs/behaviors/bluetooth.md

```diff
@@ -9,6 +9,9 @@ The bluetooth behavior allows management of various settings and states related
 between the keyboard and the host. By default, ZMK supports five "profiles" for selecting which bonded host
 computer/laptop/keyboard should receive the keyboard input; many of the commands here operation on those profiles.
 
+To enable fast switching between hosts, the keyboard will still show as "connected" on all host systems that are
+in range, though only the host bonded in the active profile will actually receive keystrokes.
```

Comment:
This isn't exactly right because it will only stay connected to any devices that have connected during the advertising period. In general the connection system works something like this:

- If the device is on an BT profile that isn't connected, it will advertise until it connects to the device on that profile
- Any other profile's host can connect while the device is advertising
- Previously connected profiles will stay connected after switching to another profile

It's a bit hard to convey what this all means in one or two sentences, so maybe this explanation is close enough? The only real incorrect piece here is that if the keyboard's current profile is connected, it won't advertise and therefore other host systems won't connect during this time even if they're in range. Thoughts?

Reply:
Sorry for the delay, I was unavailable for some days due to a family thing.

What about wording like this?

> As an implementation detail, a ZMK device may show as "connected" on multiple hosts at the same time. This is working as intended, and only the host associated with the active profile will receive keystrokes.

I think that addresses the point I was actually confused about (e.g., whether or not this behavior was a bug) without adding too many low-level details that end users aren't likely to care about.

Reply:
Yeah this seems like a great way of putting it, thanks!

Reply:
@bcat Could you submit this change, so we can get this change merged? Thanks 😄

Reply:
Whoops, lost track of this PR. Yes, done.